txnkv/transaction/pipelined_flush.go

```diff
@@ -453,6 +453,7 @@ func (c *twoPhaseCommitter) resolveFlushedLocks(bo *retry.Backoffer, start, end
 		RESOLVE_CONCURRENCY,
 		handler,
 	)
+	runner.SetStatLogInterval(30 * time.Second)
```

Comment:
It would also be helpful to log this information in `show processlist` like statements.

Reply:
This is usually async so it is not supposed to be found in `show processlist`?

Reply:
If the result is already returned it would not be displayed in `show processlist`, we need a place to view the progress of these async tasks, maybe some internal system tables like `show xxx`?

Reply:
Yeah that's a good idea. We may have a new syntax `show commit jobs` for such purposes.